projects/packages/videopress/src/client/block-editor/blocks/video/components/banner/style.scss

```diff
@@ -0,0 +1,29 @@
+.block-banner {
+	display: flex;
+    height: 48px;
+    font-size: 14px;
+    align-self: center;
+    align-items: center;
+    background: #F4F4F4;
+    padding: 0 16px;
+
+    .block-banner__content {
+        flex-grow: 2;
+        margin: 0 8px;
+    }
+
+    .components-external-link {
+        margin: 0 4px;
+        cursor: pointer;
+
+        &.is-reconnecting {
+            opacity: 0.5;
+            cursor: default;
+            text-decoration: none;
+        }
+    }
+}
+
+.editor-styles-wrapper .wp-block-post-content .components-external-link {
+    color: var( --wp-components-color-accent,var( --wp-admin-theme-color, #007cba ) );
```

Comment:
The same point about not seeing this hex on color-studio. Looking at it looks like a blue-40 or 50

Reply:
I'm not sure here. The link has the same color as the placeholder buttons:

<img width="375" alt="image" src="https://user-images.githubusercontent.com/77539/214004719-f4b5bd04-73a9-49ee-a7f1-44203486850e.png">

However, I dislike hardcoding the color :-/